front/admin_ui/poetry.lock

```diff
@@ -236,6 +236,46 @@ files = [
     {file = "audioread-3.0.0.tar.gz", hash = "sha256:121995bd207eb1fda3d566beb851d3534275925bc35a4fb6da0cb11de0f7251a"},
 ]
 
+[[package]]
+name = "boto3"
+version = "1.28.59"
+description = "The AWS SDK for Python"
+category = "main"
+optional = false
+python-versions = ">= 3.7"
+files = [
+    {file = "boto3-1.28.59-py3-none-any.whl", hash = "sha256:7623b52c135becf145f762a9cc4203a1fb30055bb1cc7a254f82e5f7954d44a1"},
+    {file = "boto3-1.28.59.tar.gz", hash = "sha256:aa861e5568a564a5ce2fff5413d6ae2cda0eed7399b3a949bc861a20915e2046"},
+]
+
+[package.dependencies]
+botocore = ">=1.31.59,<1.32.0"
+jmespath = ">=0.7.1,<2.0.0"
+s3transfer = ">=0.7.0,<0.8.0"
+
+[package.extras]
+crt = ["botocore[crt] (>=1.21.0,<2.0a0)"]
+
+[[package]]
+name = "botocore"
```

Comment:
I am wondering why `boto3` and `botocore` are added by upgrading pillow?

Reply:
I run `poetry update libcommon` at the same time, because it was not up to date

Reply:
And `s3transfer`...

Reply:
I see... I think it is because of the indirect dependency on `libcommon`...

Reply:
yes, it's what I mentioned above

> I run poetry update libcommon at the same time, because it was not up to date